pkg/statemachine/stateMachine.go

```diff
@@ -21,12 +21,14 @@ const notRunningState = ""
 //
 // usage example:
 //
-//   machine, err := NewStateMachine(states)
-//   [ err processing ]
-//   while machine.Running() {
-//     [do stuff based on machine.CurrentState(), and set direction]
-//     machine.NextState(direction)
-//   }
+// TODO: This is currently issuing linter errors if not formatted this way
```

Comment:
I actually was able to fix this in my PR. If you run `gofmt -s -w pkg/` it will fix it nicely.

Reply:
This passed: 
```
// usage example:
//
//	machine, err := NewStateMachine(states)
//	[ err processing ]
//	while machine.Running() {
//	  [do stuff based on machine.CurrentState(), and set direction]
//	  machine.NextState(direction)
//	}
```

Reply:
golangci-lint has version 1.50.1 built from 8926a95 on 2022-10-22T10:48:48Z

Reply:
OK. I suggest to leave this as-is now to have it pass, then we merge your PR on top, and hopefully that would overwrite / fix this?